### PR TITLE
5 list space availability

### LINKED ISSUE #5: 5 - list space availability
````````````````````````
As a user
So people know when my space is available
I want to able to list the dates my space is available
```

### PR DESCRIPTION
**Pull request by alextwilson**
Closes #5 

Can add available dates on space creation
Multiple dates are separated with a comma
Some controller logic extracted to view_helper

### CODE REVIEW

File: spec/features/listing_space_spec.rb [hunk 1]
```diff
@@ -26,8 +26,8 @@
     fill_in 'name', with: 'Ed\'s space'
     fill_in 'description', with: 'small'
     fill_in 'price', with: '£50'
-    fill_in 'availability', with: '13/2/2018'
+    fill_in 'availability', with: '13/2/2018, 14/2/2018, 15/02/2018'
     click_button 'List space'
-    expect(page).to have_content '13/02/2018'
+    expect(page).to have_content '13/02/2018' && '14/02/2018' && '15/02/2018'
```
Comment (Leigan0) [inline spec/features/listing_space_spec.rb]: Do any of tests check that information added to database? I know this is more unit based but we have done this for user and space

**Review by ghost — COMMENTED** [review]
🍰

**Comment by lewmoore** [comment]
:shipit:

**Comment by Leigan0** [comment]
😋

**Comment by Gleoman** [comment]
:shipit:

**Pull request merged**